modelina-website/src/components/playground/options/GeneralOptions.tsx

```diff
@@ -66,16 +66,16 @@ class GeneralOptions extends React.Component<
     }
   }
 
-  
+
 
 
   render() {
     return (
       <ul className="flex flex-col">
-        <h3 className="text-lg font-medium leading-6 text-gray-900">
+        <h3 className="py-2 w-full text-left border-b-[1px] border-gray-700 text-sm">
```

Comment:
Setting text-sm and using headers cancel them out 🙂 

```suggestion
        <h3 className="py-2 w-full text-left border-b-[1px] border-gray-700">
```

This is happening in multiple places, included in each language option, I would stick with using appropriate headers and class names 🙂